src/webchat-ui/components/history/ChatScroller.tsx

```diff
@@ -64,15 +74,38 @@ export class ChatScroller extends React.Component<InnerProps, IState> {
         this.scrollToBottom();
     }
 
+    // Add outline to the parent element when Chat log receives focus
+    handleFocus = () => {
+        if(this.rootRef.current === document.activeElement) {
+            const chatLogWrapper = this.wrapperRef.current;
+            chatLogWrapper?.setAttribute("style", "outline: 1px auto black;");
```

Comment:
How can I use a color from the theme here? @pedily

Reply:
I think it's only possible from emotion/styled right now. You can define it as a class and toggle that class.

Reply:
Thanks! Fixed